.github/workflows/java-demos.yml

```diff
@@ -35,3 +35,5 @@ jobs:
     - name: Build
       run: ./gradlew build
 
+    - name: Run filedemo
+      run: java -jar build/libs/rhino-file-demo.jar -a  ${{secrets.PV_VALID_ACCESS_KEY}} -i ../../resources/audio_samples/test_within_context.wav -c ../../resources/contexts/linux/coffee_maker_linux.rhn
```

Comment:
for some frameworks we're running demos on all platforms - it seems arbitrary that we only run on Linux for Java, .NET and Rust. We should decide one way or the other and apply to all demo actions.